src/components/Search/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+
+import Icon from '../Icons';
+
+import classnames from '../../utils/classNames';
+
+import './index.scss';
+import { IProps } from './types';
+import Field from '../Field';
+
+const baseClass = 'vant-search';
+
+const Search = ({
+  label,
+  shape = 'square',
+  background,
+  maxLength,
+  placeholder,
+  clearable = true,
+  autofocus,
```

Comment:
What does autofocus do?

Reply:
when you refresh the page, it will automatically focus on the input, you can type directly without having to mouse click the input